hera_mc/geo_handling.py

```diff
@@ -308,75 +308,71 @@ def get_ants_installed_since(self, query_date, station_types_to_check='all'):
         station_types_to_check:  list of stations types to limit check
         """
 
-        self.get_station_types(add_stations=False)
+        station_types_to_check = self.parse_station_types_to_check(station_types_to_check, add_stations=False)
         dt = query_date.gps
         found_stations = []
         for a in self.session.query(geo_location.GeoLocation).filter(
                 geo_location.GeoLocation.created_gpstime >= dt):
-            if (station_types_to_check == 'all' or
-                    self.flipped_station_types[a.station_type_name] in station_types_to_check):
+            if a.station_type_name.lower() in station_types_to_check:
```

Comment:
Note that if `parse_station_types_to_check` returned a set, this sort of check would be quicker. (Although we're talking about lists with a handful of items so it's really more about using the best tool for the job, rather than actual real-world performance.)